common/policydsl/policydsl_builder.go

```diff
@@ -28,26 +27,18 @@ var RejectAllPolicy *cb.SignaturePolicyEnvelope
 var MarshaledRejectAllPolicy []byte
 
 func init() {
-	var err error
-
 	AcceptAllPolicy = Envelope(NOutOf(0, []*cb.SignaturePolicy{}), [][]byte{})
-	MarshaledAcceptAllPolicy, err = proto.Marshal(AcceptAllPolicy)
-	if err != nil {
-		panic("Error marshaling trueEnvelope")
-	}
+	MarshaledAcceptAllPolicy = protoMarshalOrPanic(AcceptAllPolicy)
```

Comment:
nit: with the new `protoMarshalOrPanic` function, you shouldn’t need an explicit package init function. The var can simply be assigned to the result of calling the function with the result of Envelope(NOutOf(...))